submissions/benjaminKamena/phpDataase/queryDb.php

```diff
@@ -0,0 +1,33 @@
+<?php
+include('cookie.php');
+
+$results = isset($_COOKIE['customers'])? unserialize($_COOKIE['customers']) : $queryDatabase->getAllCustomers();
+
+try{
+	if(count($results) > 0){
+		echo "<table>";
+		   echo "<tr>";
+		     echo"<th>ID</th>";
+		     echo"<th>Full Name</th";
+		     echo"<th>Email</th>";
+			 echo"<th>Created At</th>";
+			 echo"<th>Actions</th>";
+		   echo"</tr>";
+    foreach($results as $row){
+		echo "<tr>";
+           echo "<td>" .$row['customer_id']."</td>";
+	       echo "<td>" .$row['customer_first_name']. " " .$row['customer_last_name']. "</td>";
+	       echo "<td>" .$row['customer_email_address']. "</td>";
+	       echo "<td>" .$row['created_at']. "</td>";
```

Comment:
Kindly add the action button as stated on the task. Check criteria number 6 of your task